### PR TITLE
Enable CFI in FMC.

### PR DESCRIPTION
**Pull request by rusty1968**
- Enable CFI protection feature in the drivers crate.
- Use CFI protection in RtAlias key derivation.

### CODE REVIEW

**Review by FerralCoder — APPROVED** [review]

File: fmc/src/main.rs [hunk 1]
```diff
@@ -44,6 +45,16 @@ pub extern "C" fn entry_point() -> ! {
         Err(e) => report_error(e.into()),
     };
 
+    if !cfg!(feature = "no-cfi") {
+        cprintln!("[state] CFI Enabled");
+        let mut entropy_gen = || env.trng.generate().map(|a| a.0);
+        CfiCounter::reset(&mut entropy_gen);
```
Comment (korran) [inline fmc/src/main.rs:51]: We should only need to do this once; we can trust that there's already some entropy in there from the ROM so it's not a big deal if somebody glitches past this...

Comment (korran) [reply fmc/src/main.rs:51]: @mhatrevi Is this even necessary? Does the ROM not leave the cfi counter in a healthy state?

Comment (rusty1968) [reply fmc/src/main.rs:51]: ROM is corrupting cfi state before handing off.

Comment (korran) [reply fmc/src/main.rs:51]: Ok, I'd like to get some more info on why we do that, but it doesn't need to block this PR.

**Review by FerralCoder — APPROVED** [review]

**Pull request merged**